package.json

```diff
@@ -41,7 +41,8 @@
     "test-ios": "./scripts/objc-test.sh test",
     "test-typescript": "dtslint packages/react-native/types",
     "test-typescript-offline": "dtslint --localTs node_modules/typescript/lib  packages/react-native/types",
-    "bump-all-updated-packages": "node ./scripts/monorepo/bump-all-updated-packages"
+    "bump-all-updated-packages": "node ./scripts/monorepo/bump-all-updated-packages",
+    "bump-oss-version": "node ./scripts/bump-oss-version.js"
```

Comment:
Why do we need two separates script? Could we have a single command that bumps everything and has a `--check` mode that verifies if there is anything to bump?

Reply:
they are two different commands:
1) `bump-all-updated-packages` -> for the other packages in the monorepo
2) `bump-oss-version` -> bumps and trigger the release of React Native

we would need to rewrite the whole thing to have 1 script that handles the logic of bumping everything. And we would still need to have the separate command for the other packages for situations like main.

Reply:
> bump-oss-version -> bumps and trigger the release of React Native

Mmm I see, this script should probably be called `trigger-react-native-release` or so, as it's more than a bump practically.

Reply:
renamed 👍 this means that once we merge this we need to also change the docs to account for the rename.
I think the order might be:
1. merge this
2. backport to 72 and 71
3. merge PR to the docs

potentially I could also do a separate PR for 0.70 branch where I just rename the `bump-oss-script` script and add it to the package.json so that the PR for the docs could be "split" in pre-70 and post-70. (which would be good considering that 70 is the last version in the support window)

Reply:
lmk and I can prep up all the PRs after we merge this and merge those all at the same time to minimize the window in which things are inconsistent

Reply:
> backport to 72 and 71

As you prefer 👍 Thanks for renaming. Probably worth to do all of them in one go and give a heads up to the release crew once this lands

Reply:
ok I can setup all the PRs and then we can import this, once this one land we land all the rest

Reply:
PRs ready:
* 72 https://github.com/facebook/react-native/pull/38888
* 71 https://github.com/facebook/react-native/pull/38890
* 70 https://github.com/facebook/react-native/pull/38892

will get to work on the one for docs asap.

Reply:
PR to docs: https://github.com/facebook/react-native-website/pull/3815